verification/curator-service/ui/README.md

```diff
@@ -34,17 +50,34 @@ US).
 - `admin3`: Optional. Meaning differs by country (e.g., this is municipality in the
 US).
 [See full definition here](https://en.wikipedia.org/wiki/List_of_administrative_divisions_by_country).
-- `gender`: Optional. One of `Female`, `Male`, `Non-binary/Third gender`, or `Other`.
-- `ageRangeStart`: Optional. Number (years) from 0 to 120.
-- `ageRangeEnd`: Optional. Number (years) from 0 to 120.
+
+##### Events
 - `dateConfirmed`: **Required**. Date (MM/DD/YYYY) on which the case was
```

Comment:
Can we do YYYY/MM/DD for dates to match the rest of the portal?

Reply:
Definitely. This is technically a matter of documentation -- either format is correctly parsed by `new Date()`, which is the format used by the data service. I'll review this with Aaron when he drafts the official doc for this.

Reply:
Gotcha. Given that data will be entered worldwide, it will be super common for people to mess it up and enter DD/MM/YYYY. If we require YYYY/MM/DD we can return an error if it's not in the right format, preventing human error.

Reply:
Makes sense. This seems like a great check to add for CSV-format/[meta-validation](https://github.com/globaldothealth/list/issues/662#issuecomment-668585762).

Reply:
Could you change the docs in the meantime?

Reply:
Sure thing, done.